lecture_4/lecture_4_solution/js/components/SpeedTyper.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import WordsContainer from "../containers/WordsContainer";
+import TypingContainer from "../containers/TypingContainer";
+import StatsContainer from "../containers/StatsContainer";
+
+const SpeedTyper = (props) => {
+  return (
+    <div className="SpeedTyper">
+      <div className="wordsbox">
```

Comment:
It's for CSS. I don't know anything about CSS so i would rather not change it, since this CSS was simply ripped